### ISSUE TITLE
[GlobalStep] Tapping a Quiz option causes its area to flick

### ISSUE THREAD

**Issue by wptester9845** [wordpress-mobile/WordPress-iOS#13328]
### Description
Tapping a Quiz option causes its area to flick.
This issue was only seen to occur on the iPad.

### Reproduction Rate
4/4 100%

### Expected behavior
No graphical issues should occur.

### Actual behavior
The Quiz area flicks when an option is pressed.

### Steps to reproduce the behavior
1. Install WordPress 14.1
2. Log in to an account.
3. Create a blog post with a quiz (the following example may be used:
```
[quiz]

[question]Insert question here
[/question]
 

[answer]Insert correct answer here
[/answer]
 

[wrong]Insert wrong answer here
[/wrong]
 

[wrong]Insert wrong answer here
[explanation]Add additional information or an explanation for why it’s wrong here[/explanation]
[/wrong]
 

[/quiz]
```
4. Visit that post.
5. Press any quiz option.
##### Tested on the following
iPad Pro 12.9” (13.1.2)

#### Please see the attached video for more information
[iOSQuizFlicking.zip](https://github.com/wordpress-mobile/WordPress-iOS/files/4134867/iOSQuizFlicking.zip)

### Submitted by:
Luis Pimenta

**Comment by designsimply** [comment]
Tested and confirmed using WPiOS 14.1.0.0 TestFlight beta on iPad Mini 4 (WiFi) (iPad5,1) iPadOS 13.3. In my testing, I checked 4 different ways of viewing/previewing and found the flickering when tapping an item on the quiz only happens in the in-app browser as seen when viewing the post from Blog Posts > View or from the editor > More menu > Preview. ([2m3s](https://cloudup.com/c7US9HozQym))

Note: there is some ongoing work happening for Previews noted in https://github.com/wordpress-mobile/WordPress-iOS/pull/13226 and which should be available to users in 14.2 (see https://github.com/wordpress-mobile/WordPress-iOS/pull/13334) so let's check back in on this issue again after that work goes live. cc @bjtitus @ScoutHarris
